data/libs/SpaceStation.lua

```diff
@@ -744,6 +882,8 @@ Event.Register("onGameEnd", function ()
 	nextRef = 0
 	equipmentStock = {}
 	equipmentPrice = {}
+	stationMarket = {}
```

Comment:
It seems `destroySystem()` has already done this

Reply:
I noticed this as well during review; this is leftover cleanup code from an earlier refactor of the file and doesn't really harm anything to clear the table twice. I'll clean it up as a small commit when I get closer to merge.

Reply:
This function duplicates most of the code from destroySystem() - I'm going to leave it as is for locality's sake.